aptos-move/framework/aptos-framework/sources/transaction_validation.move

```diff
@@ -44,7 +43,8 @@ module aptos_framework::transaction_validation {
     const PROLOGUE_EBAD_CHAIN_ID: u64 = 1007;
     const PROLOGUE_ESEQUENCE_NUMBER_TOO_BIG: u64 = 1008;
     const PROLOGUE_ESECONDARY_KEYS_ADDRESSES_COUNT_MISMATCH: u64 = 1009;
-    const PROLOGUE_EGAS_PAYER_ACCOUNT_MISSING: u64 = 1010;
+    const PROLOGUE_EFEE_PAYER_NOT_ENABLED: u64 = 1010;
```

Comment:
As mentioned in the test earlier, this needs to be properly mapped to a non-internal error similar to what I did for the new multisig account